src/attribute.ts

```diff
@@ -0,0 +1,32 @@
+import * as path from "path";
+import { Position, Range, Uri, window } from "vscode";
+import {
+  Capability,
+  isAnyCapabilityEnabled,
+} from "./capabilities/capabilities";
+import { setDecoration } from "./vscode.api";
+
+const logoPath = Uri.file(path.resolve(__dirname, "..", "small_logo-mono.png"));
+
+const type = window.createTextEditorDecorationType({
+  gutterIconPath: logoPath,
+  gutterIconSize: "70%",
+});
+
+export default function attribute(position: Position): void {
```

Comment:
I think this module deserves a bit more descriptive name + consider move this under `inlineSuggestions`